app/src/Tasks.js

```diff
@@ -34,18 +43,27 @@ const AddTask = ({ addTask }) => {
   const canAdd = task !== "";
 
   const onSubmit = (e) => {
+    const form = e.currentTarget;
+
     e.preventDefault();
     if (canAdd) {
-      addTask(task);
+      addTask(new FormData(form));
```

Comment:
Instead of sending raw JavaScript values, we send a `FormData` object, which encapsulates the file upload properly.